core/controllers/suggestion_test.py

```diff
@@ -768,6 +770,122 @@ def test_translation_suggestion_creation_with_new_images(self):
         self.assertTrue(fs.isfile('image/translation_image.png'))
         self.assertTrue(fs.isfile('image/img_compressed.png'))
 
+    def test_update_translation_updates_translation_suggestion_change(self):
```

Comment:
A suggestion is created from a different account.
Reference: https://github.com/oppia/oppia/pull/12424/files#r612927923
/cc @DubeySandeep